straxen/plugins/high_energy_processing.py

```diff
@@ -0,0 +1,130 @@
+import strax
+import straxen
+import numpy as np
+
+export, __all__ = strax.exporter()
+
+# These are also needed in peaklets, since hitfinding is repeated
+HITFINDER_OPTIONS_he = tuple([
+    strax.Option(
+        'hit_min_amplitude_he',track=False,
+        default=tuple(np.ones(800, dtype=np.int16) * 15),
+        help='Minimum hit amplitude in ADC counts above baseline. '
+             'See straxen.hit_min_amplitude for options.'
+    )])
+
+
+@export
+@strax.takes_config(
+    strax.Option(
+        'n_tpc_pmts_all',track=False,default=800
+        ),
+    *HITFINDER_OPTIONS_he)
+class PulseProcessingHe(straxen.PulseProcessing):
+    __version__ = '0.0.1'
```

Comment:
One thing which just came to my, I think we must also inherit the version number from the parent. Otherwise changes in the parent wont be visible in the child

Reply:
Okay never mind I think the best thing would be to have individual version numbers. But we should leave in the parent plugins a comment next to the version number that we also have to increase the versions of the he plugins.

Reply:
Better way is to add the parent in the depends on, then it will track changes in version of the parent in the lineage

Reply:
Okay I will add this.

Reply:
Even better is to add the version of the parent as an option

Reply:
@petergaemers , that would make it quite confusing no? You can't actually choose what version of the parent you use

Reply:
In https://github.com/AxFoundation/strax/pull/297 the version of the parent is directly retrieved from the registered plugins in the currently used context. This ensures that if you change the context you get the correct version of the parent.

Reply:
It is about tracking where the data comes from